server/channels/store/sqlstore/user_store.go

```diff
@@ -2231,3 +2253,15 @@ func (us SqlUserStore) GetUsersWithInvalidEmails(page int, perPage int, restrict
 
 	return users, nil
 }
+
+func (us SqlUserStore) RefreshPostStatsForUsers() error {
```

Comment:
Do we have/want tests for this?

Reply:
I can look into adding a test for this.

Reply:
Worst case scenario, you can do in the test something like "If driver === mysql return". But no idea if there is a better way. @isacikgoz Do you know?

Reply:
@agnivade Do you happen to know the answer to this?

Reply:
In CI, the entire test suite is run twice - once for Postgres, and once for MySQL. So it will get automatically skipped for MySQL.